docs/components/Combobox/Combobox.stories.mdx

```diff
@@ -28,19 +28,37 @@ Combobox is composed of three main parts:
 #### Trigger
 
 The trigger is the user-facing interface to open and close the Combobox and
-initiate option selection. The choice between them depends on the design and
-user experience requirements.
-
-There are two options for trigger elements of the Combobox,
-[TriggerButton](../?path=/story/components-selections-combobox-web--trigger-button)
+initiate option selection. When implementing Combobox, you may rely on the
+default trigger or, you may pass in a custom activator.
+
+The default trigger is a
+[Chip](../?path=/story/components-selections-chip-web--subtle) containing a
+label that reads "Select" and an `add` Icon suffix. The label can be customized
+by adding the optional `label` prop to better reflect the type of data being
+selected. Examples of using a default trigger can be found in the
+[single-select](../?path=/code/components-selections-combobox-web--single-select)
 and
-[TriggerChip](../?path=/story/components-selections-combobox-web--trigger-chip).
-You must have one trigger component.
+[multi-select](../?path=/code/components-selections-combobox-web--multi-select)
+stories.
+
+Notice that omitting a trigger subcomponent will result in the default trigger
+being used.
+
+The custom activator should be used when the Combobox trigger is a
+[Button](../?path=/docs/components-actions-button-docs--page) or an alternate
+variation of Chip. Button or Chip should be a child of `<Combobox.Activator />`
+and will be rendered in place of the default trigger. Examples of using a custom
+activator can be found in the
+[custom activator](../?path=/code/components-selections-combobox-web--custom-activator)
+story.
 
 #### Content
```

Comment:
Should this still be called Content?  I think so - Combobox still has a content/popover and I think we can continue to reference it like that even though `Combobox.Content` is no longer used by the consumer.

Reply:
yeah agreed. I need to read the rest of the actual text content under this heading, but referring to the stuff that pops open as "Content" is still reasonable even if we're no longer exposing that literal component